ocp_resources/migration_policy.py

```diff
@@ -0,0 +1,68 @@
+from ocp_resources.resource import Resource
+
+
+class MigrationPolicy(Resource):
+
+    api_group = Resource.ApiGroup.MIGRATIONS_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        allow_auto_converge=None,
+        allow_post_copy=None,
+        bandwidth_per_migration=None,
+        completion_timeout_per_gb=None,
+        namespace_selector=None,
+        vmi_selector=None,
+        **kwargs,
+    ):
+        """
+        Create MigrationPolicy object.
+
+        Args:
+            name (str): Migration Policy name
+            allow_auto_converge (bool, optional)
+            allow_post_copy (bool, optional)
+            bandwidth_per_migration (str, optional, i.e. "1Gi")
+            completion_timeout_per_gb (int, optional)
+            namespace_selector (dict, optional): Dict of Namespace labels selectors
```

Comment:
`namespace_selector` and `vmi_selector`
I think we need some example for the dict